edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Service/modules/RoutingModule.cs

```diff
@@ -521,6 +522,23 @@ protected override void Load(ContainerBuilder builder)
                 .Named<IClientCredentials>("EdgeHubCredentials")
                 .SingleInstance();
 
+            // ServiceIdentity "EdgeHubIdentity"
```

Comment:
Do we need to register this? Not used in other places, right? Can we move this and the code to add the identity to the tree to a separate method in Program.cs?

Reply:
I'm not sure how to get deviceId and moduleId in `Program.cs`. I think it might look a bit heavy in there...

Reply:
Hmm, that code is in dep manager. You could get it from env vars... but this is prob okay for now.